presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -155,7 +156,14 @@ else if (element instanceof LikeClause) {
 
         ConnectorTableMetadata tableMetadata = new ConnectorTableMetadata(tableName.asSchemaTableName(), ImmutableList.copyOf(columns.values()), finalProperties, statement.getComment());
 
-        metadata.createTable(session, tableName.getCatalogName(), tableMetadata);
+        try {
```

Comment:
We can simplify this and invert the check (`PrestoException` is guaranteed to have an error code, so no need to write it backwards):
```java
catch (PrestoException e) {
    if (!e.getErrorCode().equals(ALREADY_EXISTS.toErrorCode()) || !statement.isNotExists()) {
        throw e;
    }
}
```